client/deis.py

```diff
@@ -436,6 +437,9 @@ def apps_create(self, args):
 
           -b --buildpack BUILDPACK
             a buildpack url to use for this app
+
+          -r --remote REMOTE        
+            name of remote to create, default 'deis'
```

Comment:
Let's follow docopt's default syntax here:

> If you want to set a default value for an option with an argument, put it into the option-description, in form [default: <my-default-value>]

A good example is at https://github.com/deis/deis/blob/master/client/deis.py#L1342